### ISSUE TITLE
Mismatch between Documentation Example Code and Repository for 6_AspNetIdentity

### ISSUE THREAD

**Issue by ncbaihly** [IdentityServer/IdentityServer4#2429]
### Issue / Steps to reproduce the problem

On the Documentation page here: https://identityserver4.readthedocs.io/en/dev/quickstarts/6_aspnet_identity.html and in https://github.com/IdentityServer/IdentityServer4/blob/dev/docs/quickstarts/6_aspnet_identity.rst there is a mismatch in the example code given for ConfigureServices. The following line is not in the example repository but is shown in both the aforementioned places:

`services.AddTransient<IEmailSender, EmailSender>();`

**Comment by stardocs-services** [comment]
Hello, just want to ask if you managed to get 3_implicit flow authentication working?

**Comment by ncbaihly** [comment]
@stardocs-services Yes, I got 3_implicit flow authentication working. I have been working through the whole Quickstart series from the beginning.

I created this issue because from my perspective it would be nice to have the example code and the documentation matching as someone new to using IdentityServer4. I'd be happy to help contribute if the maintainers decide this is something they would like worked on.
